server/config/passport/index.js

```diff
@@ -0,0 +1,68 @@
+const LocalStrategy = require('passport-local').Strategy;
+const JWTStrategy = require('passport-jwt').Strategy;
+const ExtractJWT = require('passport-jwt').ExtractJwt;
+
+const findAdminByCredentials = require('../../queries/findAdminByCredentials');
+const findAdminById = require('../../queries/findAdminById');
+
+const getJWTSecret = require('../../utils/getJWTSecret');
+
+module.exports = (passport) => {
+  passport.use(
+    'login',
+    new LocalStrategy(
+      {
+        usernameField: 'username',
+        passwordField: 'password',
+        session: false,
+      },
+      async (username, password, done) => {
+        try {
+          // TODO not working currently
```

Comment:
is this fixed?

Reply:
Fixed in `jwt-authentication-login-page` branch